Microsoft.Toolkit.Uwp.UI.Controls/HamburgerMenu/HamburgerMenu.Properties.cs

```diff
@@ -45,6 +45,11 @@ public partial class HamburgerMenu
         /// <summary>
         /// Identifies the <see cref="PaneBackground"/> dependency property.
         /// </summary>
+        public static readonly DependencyProperty AccentColorProperty = DependencyProperty.Register(nameof(AccentColor), typeof(Brush), typeof(HamburgerMenu), new PropertyMetadata(null));
```

Comment:
This should probably default to the system accent color. It should default here, but in the xaml definition of the control.